ports/atmel-samd/common-hal/rtc/RTC.c

```diff
@@ -30,76 +30,46 @@
 #include <hal_init.h>
 #include <hpl_gclk_base.h>
 #include <hpl_pm_base.h>
-#include <hal_calendar.h>
 
 #include "py/obj.h"
 #include "py/runtime.h"
 #include "lib/timeutils/timeutils.h"
 #include "shared-bindings/rtc/__init__.h"
+#include "supervisor/port.h"
 #include "supervisor/shared/translate.h"
 
-static struct calendar_descriptor calendar;
-
-void rtc_init(void) {
-#ifdef SAMD21
-    _gclk_enable_channel(RTC_GCLK_ID, CONF_GCLK_RTC_SRC);
-#endif
-#ifdef SAMD51
-    hri_mclk_set_APBAMASK_RTC_bit(MCLK);
-#endif
-    calendar_init(&calendar, RTC);
-    calendar_set_baseyear(&calendar, 2000);
-    calendar_enable(&calendar);
-}
+// This is the time in seconds since 2000 that the RTC was started.
+static uint32_t rtc_offset = 0;
```

Comment:
Since rtc_offset is in seconds, this means that time sync with an external source will always have +-.5s error.  Perhaps as a future improvement, an additional second offset of 1/1024s could be added.

Reply:
TODO added.